package.json

```
@@ -28,7 +28,7 @@
   "homepage": "https://github.com/alastria/alastria-identity-example#readme",
   "dependencies": {
     "alastria-identity-JSON-objects": "github:alastria/alastria-identity-JSON-objects#master",
-    "alastria-identity-lib": "git://github.com/alastria/alastria-identity-lib.git#develop"
+    "alastria-identity-lib": "git://github.com/alastria/alastria-identity-lib.git#feature/update-dependencies"
```

```suggestion
    "alastria-identity-lib": "git://github.com/alastria/alastria-identity-lib.git#develop"
```
Change this, we wait to close https://github.com/alastria/alastria-identity-lib/pull/146 first

---

This change can now be applied!

---

Ready!